example/lib/main.dart

```diff
@@ -23,6 +23,10 @@ class _MyAppState extends State<MyApp> {
   String userId = 'USER_ID';
   String flagKey = 'FLAG_KEY';
 
+  final ldUser = LaunchDarklyUser(
+    privateEmail: 'example@example.com',
```

Comment:
Why not `email` here, to illustrate the point of the feature instead of `privateEmail`?

Reply:
Agree. I think I'll add a private attribute as well, so this option will be illustrated too